src/Widgets/DataImageWidget/DataImageWidgetComponent.tsx

```diff
@@ -80,5 +80,5 @@ const ImageComponent = connect(function ImageComponent({
 
   if (typeof attributeValue !== 'string') return null
 
-  return <img src={attributeValue} className={className} />
+  return <img src={attributeValue} className={className} alt="" />
```

Comment:
* [ ] FYI the component has another code path with `ImageTag`. This one should have the `alt=""`, too.

Reply:
=> https://github.com/Scrivito/scrivito-portal-app/pull/436